### PR TITLE
Week1 homework cynthia

### PR DESCRIPTION
**Pull request by cyeburu**

### CODE REVIEW

File: week-1/Homework/mandatory/5-magic-8-ball.js [hunk 1]
```diff
@@ -45,10 +45,100 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
```
Comment (illicitonion) [inline week-1/Homework/mandatory/5-magic-8-ball.js]: It looks like you made a start, but didn't quite finish this one off - can you give it another go? Was there something particular you got stuck on and need a hand with? There's always homework club if you need a little help, or you can ask questions on slack where hopefully people can help out if you're stuck :)

File: week-1/Homework/mandatory/1-syntax-errors.js [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
+addNumbers(3, 4, 6);
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  return `Hello, my name is ${name} and I am  ${age} years old`;
```
[the remark this answers is not in the document]
Comment (cyeburu) [reply week-1/Homework/mandatory/1-syntax-errors.js]: Thank you so much for taking your time to go through my code and highlighting changes that need to be made. This has been corrected

File: week-1/Homework/mandatory/2-logic-error.js [hunk 3]
```diff
@@ -1,31 +1,39 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word;
```
[the remark this answers is not in the document]
Comment (cyeburu) [reply week-1/Homework/mandatory/2-logic-error.js]: I have done the correction

File: week-1/Homework/mandatory/4-tax.js [hunk 4]
```diff
@@ -4,8 +4,11 @@
   A business requires a program that calculates how much sales tax to charge
   Sales tax is 20% of the price of the product
 */
-
-function calculateSalesTax() {}
+const salesTax = 20 / 100;
+function calculateSalesTax(ProductPrice) {
+  return `${salesTax}` * ProductPrice;
```
[the remark this answers is not in the document]
Comment (cyeburu) [reply week-1/Homework/mandatory/4-tax.js]: Your recommendation has been taken on board. Thanks

File: week-1/Homework/mandatory/5-magic-8-ball.js [hunk 5]
```diff
@@ -69,17 +89,21 @@ function shakeBall() {
...
       || answer === "Very doubtful.") {
-      return "Very negative"
+      return "very negative"
     }
-    return answer
   }
 }
+function shakeBall() {
+  console.log("The ball has shaken!");
+  return "My reply is no."
```
Comment (illicitonion) [inline week-1/Homework/mandatory/5-magic-8-ball.js]: This is generally looking great, but there's just one thing left - the magic 8 ball always returns the same thing! Let's try to always make it return a different thing - you already know how to get an element from an array, and we've seen that you can use `Math.random` to generate a random number, so how can we put the two together to get a random answer from your `ball` array?

**Pull request closed without merging**